tests/rubinrepoman_test.py

```diff
@@ -4,17 +4,66 @@
 registry, getting the list of available tags, and filtering
 that down into a list of images to pull.
 """
-
 from __future__ import annotations
 
 from typing import List
 
 from cachemachine.rubinrepoman import RubinRepoMan
-from cachemachine.types import CachedDockerImage
+from cachemachine.types import CachedDockerImage, DockerImage
 
+from .conftest import mock_registry
 from .docker_mock import DockerMock
 
-HOST = "registry.hub.docker.com"
+repo = "registry.hub.docker.com/lsstsqre/sciplat-lab"
+expected_names = {
+    "prepuller_pulled_recommended": "prepuller_pulled_recommended",
+    "recommended": "Recommended (Release r21.0.0)",
+    "r22_0_0_rc1": "Release Candidate r22.0.0-rc1",
+    "r21_0_0": "Release r21.0.0",
+    "w_2021_22": "Weekly 2021_22",
+    "w_2021_21_c0021.003": "Weekly 2021_21_c0021.003",
+    #    "w_2021_22_c0021.003_20210514": "Weekly 2021_21_c0021.003_20210514",
+    #    "w_2021_22_20210514": "Weekly 2021_21_20210514",
+    "w_2021_03": "Weekly 2021_03",
+    "d_2021_01_13": "Daily 2021_01_13",
```

Comment:
Here too, I would expect these to be `Weekly 2021 week 03` and `Daily 2021-01-13`, although I know this is pre-existing and maybe that's for another diff.

Reply:
Ah, OK, yeah, should have read farther before my last reply.  Which I'm not doing now either.

Reply:
Changing that is going to be for another diff, after we have that (and the image digest) discussion.